Tests/DistributedActorsTests/LifecycleWatchTests.swift

```diff
@@ -71,6 +71,22 @@ distributed actor Juliet: LifecycleWatch, CustomStringConvertible {
         }
     }
 
+    distributed func meetWatchAsyncCallback(
+        _ romeo: Romeo,
+        unwatch doUnwatch: Bool
+    ) async throws {
+        @Sendable
+        func asyncTerminated(_ terminatedIdentity: ClusterSystem.ActorID) async {
+            await self.probe.tell("Received terminated: \(terminatedIdentity)")
```

Comment:
This await seems not needed? Tell are unidirectional

Reply:
Get compilation errors without `await`:

```
/code/Tests/DistributedActorsTests/LifecycleWatchTests.swift:80:13: error: expression is 'async' but is not marked with 'await'
            self.probe.tell("Received terminated: \(terminatedIdentity)")
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
            await 
/code/Tests/DistributedActorsTests/LifecycleWatchTests.swift:80:13: note: property access is 'async'
            self.probe.tell("Received terminated: \(terminatedIdentity)")
```

Removing `@Sendable` gives warnings but compiles (without `await`). Should I keep `@Sendable`?

Reply:
Oh the probe access hmmm

Reply:
Checked out locally: the current code with await triggers a warning about `No 'async' operations occur within 'await' expression` removing the await "just worked" 🤔 

I think we're hitting some difference in toolchains? I'd like to try without the await and see what CI says, that'd be what I'd expect to work hmm